ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: openshift-qe-ingress-perf
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-ingress-perf-commands.sh
+  credentials:
+  - namespace: test-credentials # this entry injects the custom credential
+    name: ocp-qe-perfscale-es
+    mount_path: /secret
+    documentation: |-
```

Comment:
Need to remove this line

Reply:
@krishvoor If you want, you can add more details to the documentation line that is already at the bottom of this file. 

According to ci-operator syntax, it is expecting documentation at same level as "credentials" and "resources" in the yaml file and does not have context of "documentation" under "credentials". If you check now, you'll notice that 2 of your "mandatory" tests are failing for this reason.